components/teleport/Teleport.vue

```diff
@@ -1,86 +1,120 @@
+          <line
+            x1="30.3536"
+            y1="1.14645"
+            x2="38.3536"
+            y2="9.14645"
+            stroke="currentColor" />
+          <line
+            x1="38.3293"
+            y1="8.87629"
+            x2="30.3293"
+            y2="15.8763"
+            stroke="currentColor" />
+        </svg>
       </div>
 
-      <div class="mb-5">
-        <h3 class="has-text-weight-bold">{{ $i18n.t('teleport.to') }}</h3>
-        <TeleportTabs
-          :tabs="toTabs"
+      <div class="w-full is-relative">
+        <div class="network-title">{{ $t('teleport.destination') }}</div>
+        <NetworkDropdown
+          :options="toNetworks"
           :value="toChain"
           @select="(chain) => onChainChange(chain, false)" />
       </div>
+    </div>
 
-      <div class="mb-5">
-        {{
-          $i18n.t('teleport.receiveValue', [
-            amount || 0,
-            currency,
-            toChainLabel,
-          ])
-        }}
-        <a
-          v-safe-href="explorerUrl"
-          target="_blank"
-          rel="nofollow noopener noreferrer"
-          class="short-address">
-          {{ shortAddress(toAddress) }}
-        </a>
+    <NeoField class="mt-5">
+      <template #label>
+        <div class="font-normal">{{ $t('teleport.amount') }}</div>
+      </template>
+
+      <div class="is-relative w-full">
+        <NeoInput
+          v-model="amount"
+          root-class="w-full"
+          step="0.01"
+          type="number"
+          placeholder="Enter Amount"
+          :min="0" />
+        <div class="is-absolute-right">
+          <span
+            v-if="totalFiatValue"
+            class="token-value is-size-7 has-text-grey"
+            >~{{ totalFiatValue }} usd</span
+          >
+          {{ currency }}
+        </div>
       </div>
-
+    </NeoField>
+
+    <div
+      v-if="myBalance !== undefined"
+      class="is-size-7 is-flex is-justify-content-end is-align-items-center">
+      <span class="is-flex is-align-items-center">
+        <span class="mr-2">{{ $t('general.balance') }}:</span
+        >{{ myBalanceWithoutDivision.toFixed(4) }}{{ currency }}
+      </span>
       <NeoButton
-        :label="
-          insufficientBalance
-            ? $t('teleport.insufficientBalance', [currency])
-            : $t('teleport.send')
-        "
-        size="large"
-        class="is-size-6 submit-button"
-        :loading="isLoading"
-        :disabled="isDisabledButton"
-        variant="k-accent"
-        @click="sendXCM" />
+        no-shadow
+        rounded
+        size="small"
+        class="ml-2"
+        @click="handleMaxClick"
+        >{{ $t('teleport.max') }}</NeoButton
+      >
+    </div>
+
+    <NeoButton
+      :label="
+        insufficientBalance
+          ? $t('teleport.insufficientBalance', [currency])
+          : 'Proceed To Confirmation'
+      "
+      size="large"
+      class="is-size-6 submit-button my-5 w-full"
```

Comment:
expanded prop to make it full, missing native-type submit?

Reply:
updated